### PR TITLE
Update scheduler related notes; Environment performance tuning

### PR DESCRIPTION
**Pull request by MrGeneration**
Mainly affects source code installations

### CODE REVIEW

File: appendix/configure-env-vars.rst [hunk 1]
```diff
@@ -124,6 +124,42 @@ ZAMMAD_SESSION_JOBS_CONCURRENT
 
    Default: **unset**
 
+ZAMMAD_PROCESS_SCHEDULED_JOBS_WORKERS
+   Allows spawning an independent process just for processing scheduled jobs
+   like LDAP syncs. This can free up Zammads scheduler for other tasks when
```
Comment (mgruner) [inline appendix/configure-env-vars.rst:129]: scheduler -> background worker?

File: appendix/configure-env-vars.rst [hunk 2]
```diff
@@ -124,6 +124,42 @@ ZAMMAD_SESSION_JOBS_CONCURRENT
...
+   running tasks that require fairly long.
+
+   | Default: **unset**
+   | Maximum number of workers: ``1``
+
+   .. danger::
+
+      You can disable scheduler task by setting
```
Comment (mgruner) [inline appendix/configure-env-vars.rst:137]: disable processing of scheduled jobs by

File: appendix/configure-env-vars.rst [hunk 3]
```diff
@@ -124,6 +124,42 @@ ZAMMAD_SESSION_JOBS_CONCURRENT
...
+   though.
+
+   | Default: **unset**
+   | Maximum number of workers: ``16``
+
+   .. warning:: 🥵 **This option can be *very* CPU-intensive.**
+
+   .. danger::
+
+      You can disable scheduler task by setting
```
Comment (mgruner) [inline appendix/configure-env-vars.rst:157]: disable processing of delayed jobs by

File: install/update.rst [hunk 4]
```diff
@@ -93,6 +93,11 @@ Updating Zammad
 
    .. tab:: Source
 
+      .. danger::
+
+         Zammads scheduler script has changed! Please ensure to reinstall
```
Comment (mgruner) [inline install/update.rst:98]: Zammad's former `scheduler.rb` script has changed and is now called `background-worker.rb`.

**Review by mgruner — APPROVED** [review]
Please see some minor comments for consistency. We should avoid using the term "scheduler" now IMHO.

One question, is the wording of the variables `ZAMMAD_PROCESS_*` good in your opinion? We could still rename them at this point, e.g. to `ZAMMAD_BACKGROUND_WORKER_*`... But this would make them even longer, and I'm not sure if it would be more precise.

File: install/includes/manage-services.rst [hunk 5]
```diff
@@ -9,10 +9,10 @@ with the parent ``zammad``.
    $ # Zammad service to start all services at once
    $ systemctl (status|start|stop|restart) zammad
 
-   $ # Zammads internal railsserver
+   $ # Zammads internal puma server (relevant for display the web app)
```
Comment (YetAnotherGerrit) [inline install/includes/manage-services.rst:12]: "displaying" instead of "display"? Not 100% sure though.

**Review by YetAnotherGerrit — CHANGES_REQUESTED** [review]

File: install/update.rst [hunk 6]
```diff
@@ -93,6 +93,12 @@ Updating Zammad
 
    .. tab:: Source
 
+      .. danger::
+
+         Zammad's former ``scheduler.rb`` script has changed and is now called
```
[the remark this answers is not in the document]
Comment (MrGeneration) [reply install/update.rst:98]: The documentation always handles the latest version. In my eyes this does thus not require a version number. It technically would also apply if you'd skip Zammad 5.2 and upgrade from e.g. 5.0 to 5.3.

File: install/includes/manage-services.rst (hunk 5 above)
Comment (MrGeneration) [reply install/includes/manage-services.rst:12]: Yes you're right. 🙈

**Pull request merged**